zokrates_cli/examples/compile_errors/out_of_bounds.zok

```diff
@@ -0,0 +1,4 @@
+def main() -> field:
```

Comment:
If you're trying to test the zir optimizer here, I suspect this gets optimized away earlier. One way to prevent that is using `[a, 1] == [b, 0]` with `a` and `b` some inputs, which is reduceable to `false` only in the zir propagator